lib/smart_answer/calculators/paternity_pay_calculator.rb

```diff
@@ -12,6 +12,10 @@ def pay_duration
       paternity_leave_duration == "one_week" ? 1 : 2
     end
 
+    def paternity_deadline
```

Comment:
Could we add some tests for this method?

Reply:
Sorted 👍